js/src/modals/Verification/GatherData/gatherData.js

```diff
@@ -145,27 +149,27 @@ export default class GatherData extends Component {
   }
 
   renderCertified () {
-    const { isVerified } = this.props;
+    const { accountIsVerified } = this.props;
 
-    if (isVerified) {
+    if (accountIsVerified) {
       return (
         <div className={ styles.container }>
           <ErrorIcon />
           <p className={ styles.message }>
             <FormattedMessage
-              id='ui.verification.gatherData.isVerified.true'
+              id='ui.verification.gatherData.accountIsVerified.true'
               defaultMessage='Your account is already verified.'
             />
           </p>
         </div>
       );
-    } else if (isVerified === false) {
+    } else if (accountIsVerified === false) {
```

Comment:
it's a nullable bool. i know this is ugly, but spamming the `props` didn't feel right either. not happy with this 😕